site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -847,6 +857,12 @@ components:
             "ext":
                $ref: '#/components/schemas/Extensions'
 
+      DevicesSet:
+         type: array
+         description: A set of devices.
```

Comment:
A _set_ is an unordered collection and in this case IMHO it is misleading as the type is _array_ (which is ordered). How about _A list of devices_? Also how about _DeviceList_ instead of _DeviceSet_.

Reply:
I updated accordingly